redash/destinations/discord.py

```diff
@@ -41,15 +46,15 @@ def notify(self, alert, query, user, new_state, app, host, options):
         ]
         if alert.options.get("custom_body"):
             fields.append({"name": "Description", "value": alert.options["custom_body"]})
-        if new_state == "triggered":
+        if alert.TRIGGERED_STATE == "triggered":
```

Comment:
NO no no, we can keep `new_state`, we should replace `"triggered"` to `Alert.TRIGGERED_STATE`

Reply:
Misunderstood the suggestion. Fixed it